src/ui/wxWidgets/SystemTray.cpp

```diff
@@ -302,6 +302,7 @@ void SystemTray::ShowSetDatabaseIdDialog()
 void SystemTray::ProcessSysTrayMenuItem(int itemId)
 {
   switch(itemId) {
+    case ID_SYSTRAY_UNLOCK:
```

Comment:
The expected behavior in case of Unlock is to only unlock the database without showing the UI to the user.

Reply:
I see now, on the windows version, left-clicking the systray icon to unlock restores the window, but right-clicking and selecting the menu item does not.  What I was seeing on the Mac is that an unlock from the menu would raise the window but not display any content, tree or grid, until the window was minimized and restored.  The systray icon on the Mac displays its menu with either left or right click.  How does the 1.17.0 release behave on Linux?

Reply:
On Linux, the system tray menu appears only on a right click and I have never observed the case of an empty main frame.

Reply:
Does a left-click on the system tray icon do anything?

Reply:
A single left-click triggers an unlock and shows the main UI.

Reply:
Thanks for checking that for me.